spec/user_spec.rb

```diff
@@ -0,0 +1,9 @@
+require './lib/models/model'
+
+describe User do
+  it { is_expected.to have_property :id }
+  it { is_expected.to have_property :username }
+  it { is_expected.to have_property :password }
+
+  it { is_expected.to have_one :restaurant }
+end
```

Comment:
Did you guys get help from Thomas on this feature? I'm surprised you're doing a `Restaurant` model and a `User` model and all that. If he has been helping you, then fine. Otherwise, let's make sure to talk about this in the afternoon.